src/output/table.rs

```diff
@@ -316,7 +316,15 @@ fn determine_time_zone() -> TZResult<TimeZone> {
             }
         })
     } else {
-        TimeZone::from_file("/etc/localtime")
+        if env::consts::OS == "android" {
```

Comment:
I suppose it’s a good practice to include the “documentation” for this change.

```suggestion
        /// https://android.googlesource.com/platform/bionic/+/master/libc/tzcode/bionic.cpp
        if env::consts::OS == "android" {
```